client/src/components/Tabs.jsx

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/require-default-props */
```

Comment:
Should remove most of these or make them single liners

Reply:
removed!